docs/fieldnorm/risk_management.md

```diff
@@ -1,16 +1,35 @@
 
-HvD: note from lecture about new ISO:
-How does risk management contribute to patient safety?
-Do we need more trend analsys??
\ No newline at end of file
+Often, bioinformatics software is a part of the laboratory test, hence it should be clear which part of the laboratory test is covered by the bioinformatics risk management.  For instance, for sequencing the bioinformatics risk management starts with the initial sequencer output and ends with the storage of results.
+
+## Risk Management in Software Life Cycle Processes
+Risk management is part of the software life cycle process and is required for every (standalone) pipeline/workflow or software tool validated for diagnostics use. Risk management can be broken down into 5 steps, where step 2 and 3 should be part of the laboratories QMS. When the bioinformatics team opts for an alternative approach, step 2 and 3 need to be documented.  
+
+1. Risk Identification involves a comprehensive analysis of risk factors such as project scope, requirements, technology stack, resource availability, and external dependencies.
+Typically bioinformatics risks involve topics like:
+   - Software goal
+   - Software installation (and dependencies)
+   - Running software tools
+   - Interactions / connections with databases / other software / other hardware
+   - Data storage
+   - User errors
+   - Compute environment is down / not reachable
+   - Technical challenges
+   - Resource constraints
+2. Risk Assessment: Once risks are identified, they need to be evaluated to determine their potential impact and likelihood. This assessment allows prioritization, focusing on high-impact and high-probability risks. Risk assessment involves quantitative or qualitative analysis, aiding in the allocation of resources and attention to the most critical areas.
+3. Risk Mitigation and Planning: Based on the risks assessment, risks must be mitigated. 
+4. Monitoring and Control: Throughout the software life cycle, continuous monitoring of identified risks and the effectiveness of mitigation strategies is crucial. Often risks are already identified and discussed during development, whether or not these are all documented depends on the risks. 
+5. Communication and Documentation: Clear communication of risks, mitigation strategies, and their progress is vital among team members, stakeholders. Comprehensive documentation of identified risks, their assessment, and the implemented mitigation measures aids in transparency and knowledge transfer, ensuring a cohesive approach.
+6. Adaptation to Change: Risks and their measures may evolve over time. Risks can be reassessed, and new ones may emerge. Risk management strategies should be flexible to accommodate changes and updates, maintaining relevance throughout the software life cycle.
```

Comment:
Should it be documented by every lab when risks are reevaluated? E.g. Only when starting a new big project, or for every small bugfix, etc.

Reply:
I am interested in this as well. performing risk assessments for 1.0.0 releases is simple enough to understand, but determining at what rate a re-assessment has to be performed is challenging, especially when there are multiple bioinformaticians working on the same projects.

Reply:
Recommendation, every major + minor version or minimum two times per year?